netkan/setup.py

```diff
@@ -19,7 +19,7 @@
         'requests',
         'flask',
         'internetarchive',
-        'gunicorn>=19.9,<20.0.0',
+        'gunicorn>=19.9,!=20.0.0',
         'discord',
```

Comment:
We could probably just require >20.0.0, but it doesn't really make much difference for us.